index.html

```diff
@@ -28,9 +28,16 @@
                 <li class="menu-options"><a href="#portfolio">Portfolio</a></li>
                 <li class="menu-options"><a href="#about">About</a></li>
                 <li class="menu-options"><a href="#contact">Contact</a></li>
-                <li><img src="images/Menu.svg" alt="menu image" class="menu-image"></li>
+                <li id="burger"><img src="images/Menu.svg" alt="menu image" class="menu-image burger"></li>
             </ul>
         </nav>
+        <div id="pop-menu" class="pop-menu">
+            <ul id="menu-list">
+                <li><a  href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </div>
```

Comment:
* You have a horizontal overflow that is messing with the mobile menu dropdown mobile screens kindly make the necessary changes : 
![Screenshot (95)](https://user-images.githubusercontent.com/52497246/122839050-313c8a80-d2ef-11eb-91cd-5606b6d139ff.png)

Reply:
Hey, I think this is an issue with the browser. I have done everything I can since the beginning but the browser only resizes when it is ready. The mobile website works fine on my device and my coding partner's device. Also on my phone. I think it is the mobile emulator that is the problem. If you make your browser smaller it works or if you load the site on an actual phone it works. I have still removed the padding in the flex container. Attached are the working versions of the site. If you find an issue kindly highlight where it could have been caused in the code as I don't really understand the discrepancies in my displays. @davidosky007

![Screenshot from 2021-06-22 07-10-19](https://user-images.githubusercontent.com/72412404/122924770-bf366680-d32b-11eb-87b5-6c650aa682e3.png)

https://www.loom.com/share/6295c3530dc84ab8ada006e1ae8ac59d

Reply:
Well done 👍  you can submit it for review the next reviewer would know what to do.